src/Orchard.Web/Modules/Orchard.Modules/Controllers/AdminController.cs

```diff
@@ -132,6 +135,20 @@ public class AdminController : Controller {
                 .ToList();
             }
 
+            if (!string.IsNullOrEmpty(executionId))
+            {
+                var result = _recipeResultAccessor.GetResult(executionId);
+
+                if (result.IsSuccessful)
+                {
+                    Services.Notifier.Information(T("The recipe was executed successfully. Check the logs (recipe execution ID <strong>{0}</strong>) for more information.", result.ExecutionId));
```

Comment:
Use `.Success`.
Remove the details about this execution ID -> `The recipe was executed successfully`